Sources/TuistGenerator/Linter/SchemeLinter.swift

```diff
@@ -79,4 +80,20 @@ private extension SchemeLinter {
         let reason = "The target '\(missingTargetName)' specified in \(schemaName) code coverage targets list isn't defined in the project."
         return LintingIssue(reason: reason, severity: .error)
     }
+    
+    func projectSchemeCantReferenceRemoteTargets(schemes: [Scheme], project: Project) -> [LintingIssue] {
+        return schemes.flatMap { projectSchemeCantReferenceRemoteTargets(scheme: $0, project: project) }
+    }
+    
+    func projectSchemeCantReferenceRemoteTargets(scheme: Scheme, project: Project) -> [LintingIssue] {
```

Comment:
What about adding unit tests to make that the scheme linter is doing its job here?